proprietary/Components/src/denhaag/page-buttons.tokens.json

```diff
@@ -0,0 +1,34 @@
+{
+  "denhaag": {
+    "page-buttons": {
+      "outline-dark-action": {
+        "background-color": { "value": "{denhaag.color.white.value}" },
+        "color": { "value": "{denhaag.color.grey.4.value}" },
+        "border-color": { "value": "{denhaag.color.grey.2.value}" },
+        "min-width": { "value": "6.25rem" },
+        "margin-inline-end": { "value": "0.75rem" },
+        "margin-block-end": { "value": "0.75rem" },
+        "hover": {
+          "background-color": { "value": "{denhaag.color.white.value}" },
+          "border-color": { "value": "{denhaag.color.green.3.value}" },
+          "color": { "value": "{denhaag.color.green.3.value}" }
+        },
+        "disabled": {
+          "background-color": { "value": "{denhaag.color.white.value}" },
+          "border-color": { "value": "{denhaag.color.grey.2.value}" },
+          "color": { "value": "{denhaag.color.grey.2.value}" }
+        }
+      },
+      "icon": {
+        "margin-inline-end": { "value": "0.75rem" },
+        "max-height": { "value": "1.25rem" },
+        "max-width": { "value": "1.188rem" }
+      },
+      "option": {
+        "min-width": { "value": "2.75rem" },
+        "min-height": { "value": "2.65rem" },
```

Comment:
Is equal to 42,4px (2,65 * 16px).
There are no 0,4px, one pixel or none at all.